include/xlnt/worksheet/range_reference.hpp

```diff
@@ -56,12 +56,7 @@ class XLNT_API range_reference
     /// top_left:bottom_right.
     /// </summary>
     explicit range_reference(const char *range_string);
-
-    /// <summary>
-    /// Constructs a range reference from a pair of cell references.
-    /// </summary>
-    explicit range_reference(const std::pair<cell_reference, cell_reference> &reference_pair);
```

Comment:
just use the start/end overload below it? About the only time I'd expect to deal with a pair is a map iterator, and that would be an odd source in this case